scene/3d/navigation_mesh.cpp

```diff
@@ -490,10 +490,10 @@ void NavigationMesh::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::STRING, "geometry/source_group_name"), "set_source_group_name", "get_source_group_name");
 
 	ADD_PROPERTY(PropertyInfo(Variant::REAL, "cell/size", PROPERTY_HINT_RANGE, "0.1,1.0,0.01,or_greater"), "set_cell_size", "get_cell_size");
```

Comment:
Should we also allow for greater finetuning in `cell/size`?

Reply:
I didn't change it because baking time increases a lot for values less than 0.1. It's a sane range. There are no problems other than baking time though.

Reply:
I found this PR because i actually need greater precision on `cell/size`. Can you tweak that one too? Longer baking time is fine.